### PR TITLE
Auto update component star counts using a Github action

### PR DESCRIPTION
**Pull request by mikeesto**
This PR uses a GitHub action to update the star count of every component once a day. Closes https://github.com/svelte-society/sveltesociety.dev/issues/94 . Currently only works for components hosted on GitHub, although almost all of them are.  I wasn't sure where the best place was to put `update-stars.js` ? Thanks!

### CODE REVIEW

**Comment by kevmodrome** [comment]
This is great! I think we can put the file in a `scripts ` folder in the root.

File: scripts/update-stars.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env node
+const fs = require("fs");
+const path = require("path");
+const fetch = require("node-fetch");
+require("dotenv").config();
+
+const pathToData = path.resolve("./src/pages/components/components.json");
```
Comment (diepoe) [inline scripts/update-stars.js:7]: please use the `@/` alias instead of `./src/`

**Review by diepoe — CHANGES_REQUESTED** [review]
LGTM! Please make sure to change the requested fix, then it's fine I think.

File: scripts/update-stars.js (hunk 1 above)
Comment (mikeesto) [reply scripts/update-stars.js:7]: Hi @HAUDRAUFHAUN . Happy to change this but I can't get it to work - see below where I tried to run in a test repository. Perhaps I misunderstood what you meant?

`const pathToData = path.resolve("@/pages/components/components.json");`

`Error: ENOENT: no such file or directory, open '/home/runner/work/svelte-store-test/svelte-store-test/@/pages/components/components.json'`

Comment (diepoe) [reply scripts/update-stars.js:7]: This is weird 🤔. I think the alias maybe only works with importing the data, my fault 😅

Comment (diepoe) [reply scripts/update-stars.js:7]: But however you have to change the directory of the json file, because you moved the `update-stars.js` to scripts...

Comment (mikeesto) [reply scripts/update-stars.js:7]: The path to `components.json`? Should resolve OK. I did change the run command to the scripts folder https://github.com/svelte-society/sveltesociety.dev/pull/146/commits/1de3b66504962da6db9728b26e063eade0cc31c0#diff-7725c71eeb2eee7601fe9d992d7a0243043259fe32932cf0f3f3d191eb7e64feR21

Comment (diepoe) [reply scripts/update-stars.js:7]: Ok thanks

**Review by diepoe — APPROVED** [review]
LGTM 👍🏼

**Comment by diepoe** [comment]
I'll let @kevmodrome to final merge this because I don't now if I have access to controll the actions. Great work @mikeesto 😃

**Pull request merged**